### PR TITLE
Adds a way to check if streaming is supported

### PR DESCRIPTION
**Pull request by zachgk**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Review by frankfliu — APPROVED** [review]

File: integration/src/main/java/ai/djl/integration/tests/inference/StreamingTest.java [hunk 1]
```diff
@@ -49,6 +49,7 @@ public void testSequential() throws TranslateException {
 
             try (Predictor<Double, DoubleStream> predictor =
                     model.newPredictor(new TestTranslator())) {
+                Assert.assertTrue(predictor.supportsStreaming());
```
Comment (lanking520) [inline integration/src/main/java/ai/djl/integration/tests/inference/StreamingTest.java:52]: does python engine support streaming by default?
